data/solarsys.ssc

```diff
@@ -295,7 +302,7 @@
 		Rayleigh	[ 0.00022 0.00034 0.00070 ]
 		Absorption	[ 0.0020 0.0020 0.0030 ]
 		CloudHeight	30
-		CloudSpeed	32
+		# CloudSpeed	32
```

Comment:
don't put different updates into a single commit. and i disagree with this

Reply:
> i disagree with this

The problem about clouds moving is...well, clouds showing up in wrong locations. [Many topographical features on Earth's surface strongly influences the pattern of clouds that lie around them](https://www.youtube.com/watch?v=crad8HzMjZ0) - case in point, the Andes Mountains, which forms a unique cloud pattern that is extremely jarring to see floating anywhere else. The motion of clouds are captured well by flowmaps (not implemented), but not clouds rotating around the entire globe, and in absence of flowmaps it is better to just make the clouds stick with their parent features. The same issue also exists on Mars, where clouds in the maps form around mountains and in basins, and having them rotate independently of the surface is unrealistic.

Reply:
@JiliTheSpaceboy anyway it's better not to mix different changes in the same pull-request

Reply:
> @JiliTheSpaceboy anyway it's better not to mix different changes in the same pull-request

Yup I'm making an Issue on this one (and another one)